.github/workflows/pr.yaml

```diff
@@ -103,6 +107,16 @@ jobs:
               - '!img/**'
               - '!java/**'
               - '!notebooks/**'
+              - '!ci/cudf_pandas_scripts/**'
+            cudf_pandas:
+              - '**'
+              - 'ci/cudf_pandas_scripts/**'
```

Comment:
The `'ci/cudf_pandas_scripts/**'` line is redundant, since `'**'` already covers it. What was the intended behavior here?

Reply:
The intension of this change was to not run cpp jobs and tests if anything inside `'ci/cudf_pandas_scripts/**'` changes.

Reply:
ah, you are right. It is redundant. I'll open a fix to clean it up in `24.12`

Reply:
Sounds good! Thanks @KyleFromNVIDIA !

Reply:
I added you as a reviewer to #16713. Please take a look and make sure it got your intended changes in.